.github/release/standard/Dockerfile

```diff
@@ -3,13 +3,29 @@ FROM python:3.11-slim-bullseye
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
-RUN apt-get update && apt-get --yes --no-install-recommends install build-essential
+# Install build prerequisites.
+RUN rm -f /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+RUN \
+    --mount=type=cache,id=apt,sharing=locked,target=/var/cache/apt \
+    --mount=type=cache,id=apt,sharing=locked,target=/var/lib/apt \
+    true \
+    && apt-get update \
+    && apt-get install --no-install-recommends --no-install-suggests --yes git build-essential
 
 # Use "poetry build --format=wheel" to build wheel packages.
 COPY dist/wetterdienst-*.whl /tmp/
 
-# Install latest wheel package.
-RUN pip install --no-cache-dir $(ls -r /tmp/wetterdienst-*-py3-none-any.whl | head -n 1)[export,restapi]
+# Install package.
+# Pick latest wheel package from `/tmp` folder.
+RUN --mount=type=cache,id=pip,target=/root/.cache/pip \
+    true \
+    && pip install --upgrade pip \
+    && pip install --prefer-binary versioningit wheel \
```

Comment:
Adding `versioningit` right now will make the Dockerfile ready to use it already. I recently started using [versioningit](https://pypi.org/project/versioningit/) on one or two projects, and I love it.